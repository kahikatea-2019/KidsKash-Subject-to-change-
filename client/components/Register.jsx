import React from 'react'
import { Button } from 'react-bootstrap'
class Register extends React.Component {
  state = {

  }

  render () {
    return (
      <>
      <div className='register'>
      <h2>Please put the right format to register!</h2>
        <input />

        <input />

        <input />

        <input />

        <Button variant="info" type='button' >Register</Button>
      </div>
    </>
    )
  }
}
export default Register