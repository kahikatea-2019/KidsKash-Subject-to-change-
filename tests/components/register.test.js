import React from 'react'
import { shallow } from 'enzyme'
import Register from '../../client/components/Register'

const register = shallow(<Register />)

test('test <Register/> is set up correctly', () => {
  expect(true).toBe(true)
})

test('<Register/> has 4 input', () => {
  expect(register.find('input').length).toBe(4)
})

test('<Register/>', () => {
    expect(register.find('h2').text()).toBe('Please put the right format to register!')
  })

describe('Register', () => {
  it('returns correctly', () => {
    expect(register).toMatchSnapshot()
  })
})