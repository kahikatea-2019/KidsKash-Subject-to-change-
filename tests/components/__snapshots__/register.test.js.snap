// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register returns correctly 1`] = `ShallowWrapper {}`;
